test/test_transforms_v2_consistency.py

```diff
@@ -1259,32 +1259,17 @@ def check(self, t, t_ref, data_kwargs=None):
     def test_common(self, t_ref, t, data_kwargs):
         self.check(t, t_ref, data_kwargs)
 
-    def check_resize(self, mocker, t_ref, t):
-        mock = mocker.patch("torchvision.transforms.v2._geometry.F.resize")
-        mock_ref = mocker.patch("torchvision.transforms.functional.resize")
+    def check_resize(self, t_ref, t):
```

Comment:
Instead of checking if the dispatcher is called correctly (what doesn't happen anymore), we now check the output shape directly. This "loses" us the check if the output values actually are the same, but resize and its kernels are properly tested.

Reply:
Shouldn't we just remove this test? That the output is of the right shape is already covered in the new refactored tests. (And we should also check for consistency with V1 there, right??)

Reply:
> Shouldn't we just remove this test? That the output is of the right shape is already covered in the new refactored tests. 

Fair enough. Originally we tested `RandomResize` here, but since we cannot compare the output and even the parameter sampling is mocked for the training case, it brings very little value. I'll remove.

> And we should also check for consistency with V1 there, right??

No. So far `test/test_transforms_v2_refactored.py` does not check for consistency with v1